site/src/testHelpers/storybook.tsx

```diff
@@ -87,3 +94,45 @@ export const withDesktopViewport = (Story: FC) => (
     <Story />
   </div>
 );
+
+export const withAuthProvider = (Story: FC, { parameters }: StoryContext) => {
+  if (!parameters.user) {
+    console.warn("You forgot to add `parameters.user` to your story");
```

Comment:
Is there a way to make this fail the test outright?

Reply:
Not sure if I understand...

Reply:
This is a test helper, correct? If a developer misses something when adding a test helper, I'm not sure a `console.warn` is guaranteed to be seen.

Reply:
This is true... I will throw an error instead.